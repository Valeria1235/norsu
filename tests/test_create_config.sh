set -v

export NORSU_PATH="$PWD/pg"

# run with default options
norsu status

# check files
if [ -e "$NORSU_PATH" ]; then echo OK; fi
if [ -e "$NORSU_PATH/.norsu" ]; then echo OK; fi
if [ -e "$NORSU_PATH/.norsu.toml" ]; then echo OK; fi

# check default config
cat "$NORSU_PATH/.norsu.toml"

# remove dir
rm -rf "$NORSU_PATH"
