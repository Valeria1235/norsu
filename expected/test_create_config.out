
export NORSU_PATH="$PWD/pg"

# run with default options
norsu

# check files
if [ -e "$NORSU_PATH" ]; then echo OK; fi
OK
if [ -e "$NORSU_PATH/.norsu" ]; then echo OK; fi
OK
if [ -e "$NORSU_PATH/.norsu.toml" ]; then echo OK; fi
OK

# check default config
cat "$NORSU_PATH/.norsu.toml"
[repos]
urls = [ "git://git.postgresql.org/git/postgresql.git",]
first_match = true

[build]
configure_options = [ "CFLAGS=-g3", "--enable-cassert",]
jobs = 1

[pgxs]
default_targets = [ "clean", "install",]
default_options = []

[tools]
make = "make"

[misc]
colors = true

[commands.remove]
require_args = true

# remove dir
rm -rf "$NORSU_PATH"
